easybuild/easyconfigs/g/git-annex/git-annex-10.20230802-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'MakeCp'
+
+name = 'git-annex'
+version = '10.20230802'
+
+homepage = 'https://git-annex.branchable.com'
+description = """git-annex allows managing large files with git, without storing the file contents in git. It can sync,
+backup, and archive your data, offline and online. Checksums and encryption keep your data safe and secure. Bring the
+power and distributed nature of git to bear on your large files with git-annex."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+dependencies = [
+    ('GHC', '9.4.6', '-x86_64', SYSTEM),
+    ('GCC', '12.3.0', '', SYSTEM),
```

Comment:
wait, why the runtime dependency on a `SYSTEM` compiled `GCC`, when we're using the `GCCcore` toolchain?

Reply:
```suggestion
```

Reply:
Good question, thanks for noticing. That flew under my radar. You cannot do it without GCC (at least i get an error from Stak if there is no GCC loaded...), I just tried.
I put it instead in the GCC toolchain. I'm waiting for it to compile, I'll commit the changes once it worked on the cluster here.

Reply:
you might be able to do it with `GCCcore` if you add `binutils` as a build dependency

Reply:
Thanks for the suggestion. I tried, however unfortunately, there is still a problem with Stack 2.11.1 and GHC 9.4.6. I updated stack, but the ghc of stack gets stuck in the linker phase of GCC... are there any workarounds?

Reply:
removing `GCC` and adding `binutils` as a build dependency works for me

Reply:
to clarify, I'm happy for `GHC` to stay at the system compiler level, this is purely about the explicit `('GCC', '12.3.0', '', SYSTEM),` dependency in `git-annex`

Reply:
Sure, that makes sense, I agree.
I was already wondering why GHC is not in the GCCcore toolchain, but this was already discussed in #11310.